.github/scripts/jira/lib.ts

```diff
@@ -1,6 +1,52 @@
+import { readFile } from "fs/promises";
+import * as core from "@actions/core";
+import * as jira from "jira.js";
+import { exec } from "child_process";
+import { promisify } from "util";
+import { join } from "path";
 
-import * as core from '@actions/core'
-import * as jira from 'jira.js'
+export function generateJiraIssuesLink(label: string) {
+  // https://smartcontract-it.atlassian.net/issues/?jql=labels%20%3D%20%22review-artifacts-automation-base%3A8d818ea265ff08887e61ace4f83364a3ee149ef0-head%3A3c45b71f3610de28f429cef0163936eaa448e63c%22
+  const baseUrl = "https://smartcontract-it.atlassian.net/issues/";
```

Comment:
should we use this env here?
https://github.com/smartcontractkit/chainlink/pull/14141/files#diff-83860c228481e25b2708d5b8426dc8c07a8a9d4597c89c11ff152a44979fc7fbR403